src/app/Shared/ProgressIndicator.tsx

```diff
@@ -35,33 +35,10 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-import * as React from 'react';
-import { EventTemplate } from '../CreateRecording/CreateRecording';
 
```

Comment:
Will there be more in this file? I'm just wondering cause there is just a props type definition called LoadingPropsType inside a tsx file called ProgressIndicator and I'm not sure if those are obviously correlated.

Reply:
I'm not sure if we would do linear progress inside the button. It seems like that would either be a very small and hard to see progress bar, or a very large/wide button.

https://www.patternfly.org/v4/components/button/design-guidelines#progress-button

I'm not sure what to do about the checkmark icon design guideline here, it doesn't seem to fit in well with our specific use cases so maybe we just ignore that for now.

https://www.patternfly.org/v4/components/progress

I think we would use this component, perhaps during a file upload for example, but it would appear within the upload modal above the buttons and below the content perhaps rather than being part of the button.

![image](https://user-images.githubusercontent.com/3787464/202544738-e208d42c-7404-4e85-84da-6b7293fb494d.png)

Reply:
Niceee! This is exactly what I have in mind for multi-upload issue. Thanks for pointing out this. It would help a lot!

Reply:
Some more reading:
https://ilikekillnerds.com/2020/09/file-upload-progress-with-the-fetch-api-is-coming/
https://www.npmjs.com/package/web-streams-polyfill